lib/Player.rb

```diff
@@ -0,0 +1,34 @@
+class Player
+    attr_accessor :name, :piece
```

Comment:
Why do we need to make :name writeable? Are we changing the player's name in the middle of the game somewhere?

Reply:
I don't think you have addressed this part